tools/testing/selftests/bus1/test-io.c

```diff
@@ -371,7 +374,8 @@ int test_io(void)
 		unsigned int dests = 1UL << i;
 
 		fprintf(stderr, "it took %lu ns + %lu ns per destination for %u destinations\n",
-			base, (test_iterate(100000 >> i, dests, 1024) - base) / dests, dests);
+			base, (test_iterate(100000 >> i, dests, 1024) - base) /
+			dests, dests);
```

Comment:
This looks awful. If you really want to break the lines here, please use a temporary variable, or something.